src/UNL/Peoplefinder/Record.php

```diff
@@ -419,6 +419,10 @@ public function formatTitle()
 
     public function hasStudentInformation()
     {
+        if (!in_array('student', iterator_to_array($this->eduPersonAffiliation))) {
```

Comment:
Maybe change to make sure eduPersonAffilation is set correctly:
`if ($this->eduPersonAffiliation instanceof ArrayIterator && !in_array('student', iterator_to_array($this->eduPersonAffiliation))) {`